docs/Reference/Server.md

```diff
@@ -1865,6 +1866,7 @@ Currently the properties that can be exposed are:
   explicitly passed)
 - ignoreTrailingSlash
 - disableRequestLogging
+- createRequestLogMessage
```

Comment:
we should describe its value

Reply:
Absolutely my apologies for not including that earlier.  Is there a particular area within docs/Reference/Server.md where I should include it or does anywhere suffice?  Thank you.

Reply:
search for the docs of disableRequestLogging and put them next to it?